libyul/optimiser/Suite.cpp

```diff
@@ -310,6 +315,11 @@ void OptimiserSuite::validateSequence(string_view _stepAbbreviations)
 			nestingLevel--;
 			assertThrow(nestingLevel >= 0, OptimizerException, "Unbalanced brackets");
 			break;
+		case ':':
+			++colonDelimiters;
+			assertThrow(nestingLevel == 0, OptimizerException, "Cleanup sequence delimiter cannot be placed inside the brackets");
+			assertThrow(colonDelimiters <=1, OptimizerException, "Too many cleanup sequence delimiters");
```

Comment:
format `colonDelimiters <=1` -> `colonDelimiters <= 1`

Reply:
Good spot, shame `check_style` didn't catch this.